modin/experimental/core/storage_formats/hdk/query_compiler.py

```diff
@@ -557,18 +553,27 @@ def concat(self, axis, other, **kwargs):
 
     def drop(self, index=None, columns=None, errors: str = "raise"):
         if index is not None:
-            raise NotImplementedError("Row drop")
+            # Only column drop is supported by the HDK engine
+            raise NotImplementedError("Index drop")
```

Comment:
Maybe it just defaults to pandas?

Reply:
What do you mean?

Reply:
Will the user see this exception or not? If he sees it, then why not execute this code on pandas, and then convert it back to hdk?

Reply:
HDK automatically defaults to pandas on any `NotImplementedError` being raised: https://github.com/modin-project/modin/blob/7c1cb3c96aa1a9123fa9edb544a1cdcbe34a41f2/modin/experimental/core/storage_formats/hdk/query_compiler.py#L63

Reply:
got it